src/libpgagroal/configuration.c

```diff
@@ -2788,13 +2797,36 @@ transfer_configuration(struct configuration* config, struct configuration* reloa
    return 0;
 }
 
+static bool
+is_same_connection(struct server* s1, struct server* s2)
+{
+
+   if(!strncmp(s1->host, s2->host, MISC_LENGTH) && s1->port == s2->port)
+      return true;
+
+   return false;
+}
```

Comment:
uncrustify

Reply:
If there's an uncrustify configuration file to be used for the project I'm not able to find it.
In the meantime I've placed brackets, since indentation looks good to me (I'm using `ellemtel` codying style with 3 space indentation).

Reply:
There is a `uncrustify.sh` in the root directory with a configuration file